package task3.storage;

public class StorageException extends Exception {
    public StorageException(Throwable cause) {
        super(cause);
    }
}
